src/registry.ts

```diff
@@ -19,7 +19,11 @@ export class Registry {
   private env: Env;
 
   public constructor(public registryUrl?: string, public authToken?: string) {
-    this.init();
```

Comment:
moved more stuff into the constructor.  It helps TS know that class props are definitely set